app/models/chargeback_rate_detail.rb

```diff
@@ -175,11 +178,16 @@ def contiguous_tiers?
         error = true if tier.ends_with_infinity?
       end
 
-      break if error
+      ambiguous_error = previous_tiers_with_same_start_and_finish && consecutive_tiers?(tier, tier)
```

Comment:
@tledesma I know what you mean, tier with same start and finish is also invalid and lead to ambiguousness. I'll explain in first comment. thanks for catching this!